projects/wave-core-new/src/lib/backend/backend.model.ts

```diff
@@ -354,11 +366,10 @@ export interface MetaDataSuggestionDict {
     metaData: MetaDataDefinitionDict;
 }
 
-export interface MetaDataDefinitionDict {
-    OgrMetaData?: OgrMetaDataDict;
-}
+export type MetaDataDefinitionDict = OgrMetaDataDict;
 
 export interface OgrMetaDataDict {
+    type: 'ogrMetaData';
```

Comment:
Maybe we should allow a capital beginning for Operators and Types…

Reply:
everything I changed is a type. So you want `"type": "LinearGradient"` and `"type": "Internal"` capitalized?